ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/tracing/vm/VmTraceGenerator.java

```diff
@@ -172,6 +180,10 @@ private void handleDepthIncreased(
         } else {
           if (currentTraceFrame.getPrecompiledGasCost().isPresent()) {
             op.setCost(op.getCost() + currentTraceFrame.getPrecompiledGasCost().get().toLong());
+          } else if (currentOperation.equals("STATICCALL")
+              && nextTraceFrame.map(TraceFrame::getDepth).orElse(0)
```

Comment:
Have you tested with several levels deep ?

Reply:
I added a new test transaction with several levels deeps :  

- Block 0x17 with depth == 0
- Block 0x18 with depth == 2

Reply:
Nice